smarts/core/smarts.py

```diff
@@ -608,7 +608,6 @@ def _pybullet_provider_sync(self, provider_state: ProviderState):
                         pybullet_vehicle = self._vehicle_index.vehicle_by_id(vehicle_id)
                         assert isinstance(pybullet_vehicle.chassis, BoxChassis)
                         pybullet_vehicle.update_state(vehicle, dt=dt)
-                        pybullet_vehicle.updated = True
```

Comment:
This line was also a (relatively-minor) bug.  It should have been `vehicle.updated = True`.